i18n/eo.yml

```diff
@@ -0,0 +1,25 @@
+- id: prev_page
+  translation: "« paĝo"
```

Comment:
Can you remove these arrows? The template adds arrows, we don't need to add them in translation files

Reply:
Done